resource.go

```diff
@@ -12,8 +12,9 @@ const (
 )
 
 type Config struct {
-	ClusterStatusFunc func(obj interface{}) (providerv1alpha1.StatusCluster, error)
+	ClusterStatusFunc func(v interface{}) (providerv1alpha1.StatusCluster, error)
```

Comment:
is `obj` to `v` a codestyle decision?

Reply:
Yeah. Pretty random though but I noticed most interface values are just called `v`. We made up this `obj` thing once in `operatorkit` but this seems to be odd to me now.